docs/manage/fleet/_index.md

```diff
@@ -47,6 +47,16 @@ Role Based Access Control (RBAC) ia a way to enforce security in the [Viam app](
 
 A user can have one or more roles, granting the user the respective permissions of each role.
 
+## Robots
```

Comment:
Seems like this should be pared down since it's exactly the same as what is in the /robots page.

Also why is this below permissions? It seems like it should be above locations since in locations it says every robot belongs to a location

Reply:
going to do that with the fleet reorg PR I think - that moves everything even more